fem/tbilinearform.hpp

```diff
@@ -101,13 +115,13 @@ class TBilinearForm : public Operator
         solVecLayout(sol_fes),
         int_rule(),
         coeff(integ.coeff),
-        assembled_data(NULL),
+        assembled_data(),
         in_fes(sol_fes)
-   { }
+   { assembled_data.Reset(MemoryType::HOST_32); }
```

Comment:
The memory type here should depend on `SS` which is the same as `impl_traits_t::simd_size`.

Reply:
`done`